lib/rules/web/admin_console/4.2/deployment.xyaml

```diff
@@ -22,3 +22,13 @@ confirm_cancel_rollout:
   params:
     button_text: Yes, cancel
   action: click_button
+deploy_image_from_image_registry:
+  action: goto_deploy_image_page
+  action: search_image
+search_image:
```

Comment:
We can pass parameter to `set_input_value`

Reply:
Better to add a clear input action before input and search